Modules/Loadable/Segmentations/EditorEffects/qSlicerSegmentEditorPaintEffect.cxx

```diff
@@ -944,7 +944,7 @@ QIcon qSlicerSegmentEditorPaintEffect::icon()
 }
 
 //---------------------------------------------------------------------------
-QString const qSlicerSegmentEditorPaintEffect::helpText() const
+const QString qSlicerSegmentEditorPaintEffect::helpText() const
```

Comment:
In a follow-up commit, we will look into changing `const QString` to `QString` as it is meaningless because of `return-by-value`

Reply:
I think that const on the return type is meaningless and ignored for `return-by-value`.  If I remember correctly, the signature of the function is not changed because the return const has been ignored by the compiler (this assumption may not be true for all compilers).